### ISSUE TITLE
Norns crashes every time when loading last saved project

### ISSUE THREAD

**Issue by GordeyChernyy** [itsyourbedtime/orca#17]
When I saved my project and closed the app then tried to launch Orca again Nornes freezes with message Loading...

After restarting I can't launch Orca again. The only way is to to delete saved project files from file browser

I am using Orca from Maiden

**Comment by GordeyChernyy** [comment]
I think I have similar issue https://github.com/itsyourbedtime/orca/issues/14#issue-774885985 
How to capture logs? Maiden clears all logs on crash

**Comment by GordeyChernyy** [comment]
@frederickk

**Comment by frederickk** [comment]
Yup, this is a known issue. Tracking it down has been challenging.

**Comment by GordeyChernyy** [comment]
Let me know if I can help to make repro steps or capture logs. I found a workaround by deleting a preset file that is associated with the .orca file before launching the app
